bindata/network/kuryr/006-controller.yaml

```diff
@@ -0,0 +1,63 @@
+# More info about the template: https://docs.openstack.org/kuryr-kubernetes/latest/installation/containerized.html#generating-kuryr-resource-definitions-for-kubernetes
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kuryr-controller
+  namespace: kuryr
+  annotations:
+    kubernetes.io/description: |
+      This deployment launches the kuryr-controller component.
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: kuryr-controller
+      labels:
+        app: kuryr
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      serviceAccountName: kuryr
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      containers:
+      - image: {{ .ControllerImage }}
+        name: controller
+{{ if (default .ControllerEnableProbes true) eq "true" }}
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: {{ default .ControllerProbesPort 8082 }}
+            scheme: HTTP
+          timeoutSeconds: 5
+        livenessProbe:
+          httpGet:
+            path: /alive
+            port: {{ default .ControllerProbesPort 8082 }}
+          initialDelaySeconds: 15
+{{ end }}
+        # FIXME(dulek): This shouldn't be required, but without it selinux is
```

Comment:
weird!

Reply:
Yeah, but we've gone through that in openshift-ansible, so copying the workaround here for now.